### DISCUSSION TITLE
Friends on leaderboards are not highlighted

### DISCUSSION THREAD

**Discussion by mixalisnew** [ppy/osu#27451] [Ideas] [answered]
As the tittle saus on lazer while on the song select your friends are not being highlighted on the leaderboards and its hard to spot them. Probably this is just something that got overlooked is my guess.

**Comment by Gabixel** [comment, marked as answer]
Note that the entire song selection screen is getting a redesign, so for now it got _intentionally_ overlooked!

**Reply by peppy** [reply]
did it?

**Reply by mixalisnew** [reply]
so that means someone forgot to add that to the to do list? xD

**Reply by peppy** [reply]
No, it just hasn't happened yet. Along with 10394832901 other things
